structcol/refractive_index.py

```diff
@@ -158,9 +159,202 @@ def n_silica_colloidal(w):
 def n_water(w):
     return 1.33
 
-def n_cargille(i,series,wavelength):
-    # where i is the cardinal number of the liquid (starting with 0) and w is the wavelength, in nm (converted to um)
-    return 1.45
+def n_cargille(i,series,w):
```

Comment:
The way that comes to mind for me to put the Cargille oils into the dictionary is to use a nested for loop to iterate through the oil series and the oil cardinal numbers, adding a new key and value to the dictionary on each iteration, where the value would be the index that is returned by the n_cargille() function. This seems a little inefficient and also means that we will be keeping the n_cargille() function and adding to the dictionary, but do you think it's an okay solution?